doc/reference/views.rst

```diff
@@ -1023,6 +1023,9 @@ attributes:
   the list view)
 ``class``
   adds HTML classes to the root HTML element of the Kanban view
+``examples``
+  shows examples link on quick create of column, when clicked examples will
+  be displayed in popup
```

Comment:
if set to a key in the [KanbanExamplesRegistry](https://github.com/odoo/odoo/blob/99821fdcf89aa66ac9561a972c6823135ebf65c0/addons/web/static/src/js/views/kanban/kanban_examples_registry.js), examples on column setups will be available in the grouped kanban view. [Here](https://github.com/odoo/odoo/blob/99821fdcf89aa66ac9561a972c6823135ebf65c0/addons/project/static/src/js/project_task_kanban_examples.js#L27) is an example of how to define those setups.

Reply:
Done